packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/configuration.md

```diff
@@ -1,8 +1,33 @@
+## Verify that the Interaction Code grant type is enabled
+
+<ApiLifecycle access="ea" />
+
+The Interaction Code grant type is a sign-in flow used by embedded applications to connect to Okta. The **Embedded widget sign-in support** setting controls this grant type for all OpenID Connect and API access management servers in your org. With this setting enabled, you can allow or deny apps the ability to use embedded sign-in flows across your entire org. See [Interaction Code grant type](/docs/concepts/interaction-code/) for detailed information on this grant type and how to use it.
+
+> **Note:** Super Admin permissions are required. If you don’t have those permissions for your org, contact your administrator.
```

Comment:
Should we link to the administrator roles docs in h.o.c.? I don't know how familiar our average developer-as-admin users are with the various roles.

Reply:
https://help.okta.com/oie/en-us/Content/Topics/Security/administrators-super-admin.htm